ideascube/serveradmin/catalog.py

```diff
@@ -344,10 +367,23 @@ class StaticSite(SimpleZipPackage):
     typename = 'static-site'
     handler = Nginx
 
+    # [FIXME] This propertie looks like hacks.
+    # We should rewrite few templates to have just one able to handle all
+    # static sites.
+    @property
+    def template_id(self):
+         if self.id == 'w2eu':
+             return self.id
+         base_name, extension = self.id.rsplit('.', 1)
```

Comment:
We may have language specific maps soon (like `jordan.map.ar`, `jordan.map.en`…), even if this is a "temporary hack", I'd use something like `if ".map" in base_name` to handle this.